air/src/stack/mod.rs

```diff
@@ -305,6 +306,10 @@ trait EvaluationFrameExt<E: FieldElement> {
     fn stack_item_next(&self, index: usize) -> E;
     /// Gets the current element of the fmp register in the trace.
     fn fmp(&self) -> E;
+    /// Gets the next element of the fmp register in the trace.
+    fn fmp_next(&self) -> E;
+    /// Gets the current value at the specified index in the decoder helper register.
```

Comment:
nit: I'd change this comment to "Gets the current value of user op helper register located as the specified index."